From: Jakub Adam <jakub.adam@ktknet.cz>
Date: Sun, 8 Jul 2012 15:36:59 +0200
Subject: find-widget-export-actions-when-attaching

The library search path is hand over to the forked process by setting
GAMMARAY_PROBE_PATH environment variable. This doesn't work when Gammaray
attaches to a running Qt application. As we know the exact location on
a Debian system, we don't have to rely on env variables and use
hardcoded GAMMARAY_LIB_INSTALL_DIR instead, which works in all cases.
---
 core/tools/widgetinspector/widgetinspector.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/tools/widgetinspector/widgetinspector.cpp b/core/tools/widgetinspector/widgetinspector.cpp
index 1fafc92..234933c 100644
--- a/core/tools/widgetinspector/widgetinspector.cpp
+++ b/core/tools/widgetinspector/widgetinspector.cpp
@@ -246,7 +246,7 @@ void WidgetInspector::callExternalExportAction(const char *name,
 {
   if (!m_externalExportActions.isLoaded()) {
     const QString probePath =
-      QString::fromLocal8Bit(qgetenv("GAMMARAY_PROBE_PATH") + "/gammaray");
+      QString::fromLocal8Bit(GAMMARAY_LIB_INSTALL_DIR "/gammaray");
 
     m_externalExportActions.setFileName(
       probePath + QLatin1String("/libgammaray_widget_export_actions"));
